spec/services/api_login_manager_spec.rb

```diff
@@ -24,60 +25,87 @@
       end
 
       it "update user's auth_token" do
-        skip 'Una vez que definan a user, este test va a fallar. ¿Por qué?'
-        expect { subject }.to change { user.auth_token }
+        expect { subject }.to change { user.reload.auth_token }
       end
 
       it 'returns the auth_token' do
-        skip 'Implementar'
+        expect(subject).to eq(user.reload.auth_token)
       end
     end
 
     context 'when no email is provided' do
-      subject {  } # implementar
+      subject { described_class.new(password: password) }
 
       it 'returns false' do
-        skip 'Implementar'
+        expect(subject.call).to be_falsey
       end
 
       it 'returns EMPTY_EMAIL error with a reader' do
-        skip 'Implementar'
+        subject.call
+        expect(subject.error).to be(ApiLoginManager::EMPTY_EMAIL)
       end
     end
 
     context 'when no password is provided' do
-      subject {  } # implementar
+      subject { described_class.new(email: user.email) }
 
       it 'returns false' do
-        skip 'Implementar'
+        expect(subject.call).to be_falsey
       end
 
       it 'returns EMPTY_PASSWORD error with a reader' do
-        skip 'Implementar'
+        subject.call
+        expect(subject.error).to be(ApiLoginManager::EMPTY_PASSWORD)
       end
     end
 
     context 'when the email is incorrect' do
-      subject {  } # implementar
+      subject do
+        described_class.new(
+          email: Faker::Internet.unique.email,
+          password: user.password
+        )
+      end
 
       it 'returns false' do
-        skip 'Implementar'
+        expect(subject.call).to be_falsey
       end
 
       it 'returns USER_NOT_FOUND error with a reader' do
-        skip 'Implementar'
+        subject.call
+        expect(subject.error).to be(ApiLoginManager::USER_NOT_FOUND)
       end
     end
 
     context 'when the password is incorrect' do
-      subject {  } # implementar
+      subject do
+        described_class.new(email: user.email, password: SecureRandom.uuid)
+      end
 
       it 'returns false' do
-        skip 'Implementar'
+        expect(subject.call).to be_falsey
       end
 
       it 'returns WRONG_PASSWORD error with a reader' do
-        skip 'Implementar'
+        subject.call
+        expect(subject.error).to be(ApiLoginManager::WRONG_PASSWORD)
+      end
+    end
+
+    context 'when ExternalValidator fails' do
+      before do
+        allow(ExternalValidator).to receive(:call).and_return(false)
+      end
+
+      subject { described_class.new(email: user.email, password: user.password) }
+
+      it 'returns false' do
+        expect(subject.call).to be_falsey
+      end
+
+      it 'returns EXTERNAL_VALIDATOR error with a reader' do
+        subject.call
+        expect(subject.error).to be(ApiLoginManager::EXTERNAL_VALIDATOR)
       end
     end
```

Comment:
Creo que modificaría los `be_falsey` por `be false`, por el mismo comentario que hice anteriormente.